plugins/modules/nmcli.py

```diff
@@ -1226,10 +1226,7 @@
 # second_tenant_ip: "204.0.113.77/23"
 # ```
 
-
-
 ## playbook-add.yml example
-
 - hosts: openstack-stage
```

Comment:
Can we keep at least some of the empty lines above? I.e. two empty lines between the commented out inventory and the example comment, and one empty line between the example comment and the begin of the playbook?

Reply:
Keep or add? I did not touch the part between the inventory and the example. And there is one empty line between the example and the playbook.

Reply:
Keep the lines that this PR is removing, or at least some of them. Rigth now it removes some lines, and now it re-adds some others that weren't removed before.

Reply:
![image](https://github.com/user-attachments/assets/a44a2f42-851b-4740-a39c-1ab662a50bad)

Reply:
Technically, that's not an inventory :-P But OK, I got the idea.

Reply:
@felixfontein there are two empty lines between them, I believe that should be OK. Please let me know otherwise.